index.html

```diff
@@ -289,6 +289,33 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+<section class="contact-me">
+  <div>
+  <p class="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+  </div>
+  <div>
+  <form action="https://formspree.io" method="post">
+  <input type="text" id="name" name="name" placeholder="Full Name" maxlength="30" required>
+  <input type="email" id="email" name="email" placeholder="Email Addres" required>
+  <textarea name="textarea" placeholder="Enter text here" required></textarea>
+  <div><button id="footer-button" type="submit">Get in touch</button></div>
```

Comment:
[OPTIONAL] Kindly fix the Button interactions such as hover, active (when it's pressed), disabled and enable (default state) to match the Figma template's design. 

Please **NOTE** that this is a key requirement as per the project requirements though highlighted as optional, however your project will still be  be approved